fix: typo on app setting save (#35733)
RocketChat/Rocket.Chat: .changeset/witty-foxes-thank.md

---
"@rocket.chat/meteor": patch
"@rocket.chat/i18n": patch
---

> Fixes a typo whenever an app setting was updated.

Reviewer: ```suggestion
Fixes a typo in the app update success toast
```
